wallet/src/de/schildbach/wallet/ui/send/SendCoinsInternalActivity.java

```diff
@@ -1,71 +0,0 @@
-/*
- * Copyright 2020 Dash Core Group
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package de.schildbach.wallet.ui.send;
-
-import android.app.Activity;
-import android.content.Context;
-import android.content.Intent;
-import android.nfc.NfcAdapter;
-import android.os.Bundle;
-
-import de.schildbach.wallet.data.PaymentIntent;
-
-public class SendCoinsInternalActivity extends SendCoinsActivity {
```

Comment:
This activity does not seem to serve any purpose that cannot be included into the `SendCoinsActivity`